mobile/library/cc/engine_builder.cc

```diff
@@ -857,8 +852,6 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
     (*reloadable_features.mutable_fields())[guard_and_value.first].set_bool_value(
         guard_and_value.second);
   }
-  (*reloadable_features.mutable_fields())["prefer_quic_client_udp_gro"].set_bool_value(
```

Comment:
We'd discussed leaving this in in case import had concerns?  If you want to remove it today let's either get pre-approval from EPT that they don't have concerns, or you'd have to monitor import to make sure if they want to do delay rollout we add a platform-switch instead of a disable on import.

Reply:
I won't insist on keeping the existing API because we have an mobile API to flip a reloadable flag now. We can explicit set the flag there if needed.